.pip.canonical-webteam-jp.ubuntu.com-e040a72a.hash

```diff
@@ -0,0 +1 @@
+640290c1-d864b255
```

Comment:
Please remove and `git ignore` these files (I don't know why they're not already ignored)